config.yaml

```diff
@@ -1,8 +1,59 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: honeycomb-writekey
+  namespace: kube-system
+type: Opaque
+data:
+  key: <BASE-64-ENCODED-HONEYCOMB-WRITEKEY>
 ---
-apiHost: https://api.honeycomb.io
-writekey: "YOUR_HONEYCOMB_WRITEKEY_HERE"
-parsers:
-- dataset: kubernetestest
-  parser: json
-
-verbosity: debug
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: honeycomb-agent-config
+  namespace: kube-system
+data:
+  config.yaml: |-
+    apiHost: https://api.honeycomb.io
```

Comment:
I'm not sure this will make it into the Th deadline, but it would be nice to use Jsonnet here too. Maybe a future work item?